src/metatensor/models/utils/compute_loss.py

```diff
@@ -61,6 +62,9 @@ def compute_model_loss(
             for system in systems
         ]
         # Create new "displaced" systems:
+        logging.warning(
+            "Systems are recreated, which may cause loosing the neighborlists."
```

Comment:
How bad is this? @Luthaf

Reply:
What I noticed, is I can't currently train on stresses because of this feature, since I loose the neighborlist. I can in principle create a workaround for this and attach the old NLs to the systems created here